docs/IE_DG/Int8Inference.md

```diff
@@ -1,15 +1,35 @@
 # Low-Precision 8-bit Integer Inference {#openvino_docs_IE_DG_Int8Inference}
 
-## Table of Contents
-1. [Supported devices](#supported-devices)
-2. [Low-Precision 8-bit Integer Inference Workflow](#low-precision-8-bit-integer-inference-workflow)
-3. [Prerequisites](#prerequisites)
-4. [Inference](#inference)
-5. [Results analysis](#results-analysis)
+## Introduction
+The goal of this document is to describe some aspects of low-precision inference using OpenVINO Inference Engine (IE). 
+Currently, many Intel HW devices support 8-bit arithmetic out-of-the-box. It allows boosting inference performance 
+significantly by loading more data into a single processor instruction and executing more arithmetical operations at
+one time. The real performance improvement from 8-bit inference depends on the HW but theoretically it can achieve **4X**
+ compared with FP32 execution and **2X** with FP16 at some cost of accuracy which is negligible in many cases.
```

Comment:
Is it really needed: "at some cost of accuracy which is negligible in many cases"? This is not about the quantization but rather FP vs INT8 here and I think it's useless to talk about the accuracy here.